docs/trestle_author.md

```diff
@@ -18,347 +27,573 @@ While trestle provides editing support for OSCAL there is an unfortunate truth t
+    To validate the documents against the template run:
 
-e.g.:
+    > trestle author folders template-validate -tn my_task_2
 
-```text
-trestle_root
-┣ .trestle
-┃ ┣ author
-┃ ┃ ┗ my_task_2
-┃ ┃ ┃ ┣ template.md
-┃ ┃ ┃ ┗ template.drawio
-┃ ┗ config.ini
-```
+    This will ensure that the respective template files are parseable.
+
+    #### Extra options
+
+    - This command has no extra options
+
+    ### Validating the documents against templates
+
+    The validation in `trestle author folder` runs similarly as in `docs` but now each document will be validated against the template with the same name as in the template folder.
+
+    To validate the documents against their respective templates, run:
+
+    > trestle author folders validate -tn my_task_name
+
+    This will validate all files. Please note that all files from the template folder must be present in the individual document folders.
+
+    #### Extra options
+
+    - If `--governed-heading` (`-gh`) is passed it will ensure that the governed content was not modified. Governed content comes in the **key:** value format:
+      ```markdown
+         # Governed section
+         **Content Type:**  Foo
+         **Author(s):**  Bah
+         **Executive Owner:**  Stuff
+         **Technical Approver:**   John Doe (approved)
+         **Version:** 1.0.1
+      ```
+      Running `trestle author docs validate -tn docs_task -gh="Governed section"` will ensure this content is present in the document.
+    - If `--ignore ^_.*` (`-ig`) is passed it will validate all files except folders and files that start with underscore `_`. Use this option when you would like to ignore any folders or files that match given regular expression.
+    - If `--header-validate` (`-hv`) is passed the header will be validated as well.
+    - If `--header-only-validate` (`-hov`) is passed only the header and NOT the body will be validated.
+    - If `--readme-validate` (`-rv`) is passed README.md will be validated as well, otherwise it is ignored.
+    - If `--recurse` (`-r`) is passed the documents in the subfolders will also be validated. By default `author docs` only indexes a flat directory.
+    - If `--template-version 1.0.0` (`-tv`) is passed the header field `x-trestle-template-version` will be ignored and document will be forcefully validated against template of version `1.0.0`.
+      Use this for testing purposes _only_ when you need to validate the document against a specific template. By default the template version will be determined based on `x-trestle-template-version` in the document.
+
+    </details>
+
+  - <details>
+      <summary><b>headers - Validate documents headers only against a global or a specific template </b></summary>
+
+    ## Validate documents headers only against a global or a specific template
+
```

Comment:
This is hard to parse.  Do you mean: Validate only the document headers, and do it against a global or specific template.

I would avoid "documents headers" if you do mean "document headers" - and putting "only" first makes it clear only the headers are validated.

Reply:
Yes, fixed to Validate only the headers...